change-notes/1.18/analysis-javascript.md

```diff
@@ -108,6 +108,7 @@
 | Missing rate limiting | More true-positive results, fewer false-positive results | This rule now recognizes additional rate limiters and expensive route handlers. | 
 | Missing X-Frame-Options HTTP header | Fewer false-positive results | This rule now treats header names case-insensitively. |
 | Reflected cross-site scripting | Fewer false-positive results | This rule now treats header names case-insensitively. |
+| Semicolon insertion | Fewer results | This rule now ignores TypeScript files as it did not work correctly. |
```

Comment:
Perhaps also mention the other rules? Also, we may want to emphasise that they are only disabled temporarily.